app/lib/finder_api.rb

```diff
@@ -0,0 +1,43 @@
+require 'delegate'
+
+class FinderApi
+  def initialize(content_api:, search_api:)
+    @content_api = content_api
```

Comment:
Are we happy with calling this `content_api` when it should be the `content_store`?